IrisSoftware/main.py

```diff
@@ -30,8 +31,8 @@ def __init__(self) -> None:
         # END TODO
 
         # Classes & objects
-        self.camera = cv2.VideoCapture(0)
-        self.ui = UI()
+        self.camera = Camera()
+        self.ui = UI(self.camera.getResolution())
```

Comment:
Do we need a whole Camera class if the only functionality used is a thin wrapper over camera.read(), used twice and `capture.get(cv2.CAP_PROP_FRAME_WIDTH)`? We could simplify our file structure by eliminating the Camera class and just inlining the calls to get width and height when constructing a UI, and replacing the `getFrame()` method calls with `_, frame = self.camera.read()`

Reply:
While we could simplify our file structure, I think this keeps our code just a little bit cleaner. Encapsulating the camera resolution and `getFrame` into a class allows our `IrisSoftware` class to be a little leaner, particularly in `__init__`. My thinking was to make our `IrisSoftware` class as simple as possible and keep separate components of our application in their own little space. This should also, if we decide to add unit tests, make writing unit tests easier.

Reply:
I guess it's such a simple abstraction, it doesn't hurt too much either way. Although, it might be better to rename `Camera.py` to `camera.py` to match the rest of our files.